app/views/layouts/_asset_reconciliation.html.erb

```diff
@@ -0,0 +1,60 @@
+<meta name="page-cached-at" content="<%= Time.current.to_i %>">
+
+<meta name="proper-stylesheet-crayons" content="<%= stylesheet_url "crayons" %>">
+<meta name="proper-stylesheet-views" content="<%= stylesheet_url "views" %>">
+<meta name="proper-stylesheet-minimal" content="<%= stylesheet_url "minimal" %>">
```

Comment:
These are meta tags which _store_ the location of the proper stylesheet URLs, but don't do anything unless we insert them with JS below.